docker-compose.yml

```diff
@@ -16,6 +16,8 @@ services:
           memory: 128M
     environment:
       - REDIS_HOST=redis
+      - FASTCGI_PASS_HOST=phpfpm
```

Comment:
Don't name variables thinking to their use, but to their content. I mean `PHPFPM_HOST` describes the content of the variable, the current name describes the intended usage ... which is wrong IMHO